mango-cli/src/main/scala/org/bdgenomics/mango/cli/VizReads.scala

```diff
@@ -843,6 +843,10 @@ class VizReads(protected val args: VizReadsArgs) extends BDGSparkCommand[VizRead
     VizReads.server.setHandler(handlers)
     handlers.addHandler(context)
     VizReads.server.start()
+
+    // warm up reads data by doing small query to force one-time dataset partition discovery scan
+    VizReads.readsCache = VizReads.materializer.getReads().get.getJson(ReferenceRegion("chr1", 20000000L, 20000100L))
```

Comment:
This should only happen if the files are partitioned, and should go in AlignmentRecordMaterialization. We should call a .first instead because not all references have chr1